frontend/smart-kickers-game/src/tests/App.spec.js

```diff
@@ -0,0 +1,43 @@
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import WS from 'jest-websocket-mock';
+import { getElementWhichContain } from './helpers';
+import App from '../App';
+
+let ws;
+describe('<App />', () => {
+  beforeEach(() => {
+    ws = new WS('ws://localhost:3006/csc');
+  });
+
+  afterEach(() => {
+    WS.clean();
+  });
+
+  it('should render correctly', async () => {
+    render(<App />);
+
+    expect(getElementWhichContain('Blue:')).toBeDefined();
+    expect(getElementWhichContain('White:')).toBeDefined();
+  });
+
+  it('should update score on score message', async () => {
+    render(<App />);
+    await ws.connected;
+
+    ws.send(JSON.stringify({ blueScore: 1, whiteScore: 1 }));
+
+    expect(getElementWhichContain('Blue:')).toHaveTextContent('1');
+    expect(getElementWhichContain('White:')).toHaveTextContent('1');
+  });
+
+  it('should update score on score message', async () => {
+    render(<App />);
+    await ws.connected;
+
+    ws.send(JSON.stringify({ blueScore: 10, whiteScore: 14 }));
+
+    expect(getElementWhichContain('Blue:')).toHaveTextContent('10');
+    expect(getElementWhichContain('White:')).toHaveTextContent('14');
+  });
+});
```

Comment:
These two tests are doing the same thing but with another score, mby make one test with for example blue: 1 and white 14?

Reply:
Good point, for now single test should be enough. In that case I'll remove that with 1 in both score.